virttest/tests/unattended_install.py

```diff
@@ -406,6 +406,10 @@ def answer_kickstart(self, answer_path):
             else:
                 contents = dummy_graphical_re.sub('graphical', contents)
 
+        if self.params.get("cmd_only_use_disk"):
+            insert_info = self.params.get("cmd_only_use_disk") + '\n'
+            insert_pos = "install\n"
```

Comment:
If we locate the cmd_only_use_disk cmd by "install\n" key word. I think we will have risk that the "install\n" keyword may appear in several places in ks file. And, if "install\n" removed in ks file . the cmd_only_use_disk will  not added in.
So I think it maybe better to remove the dependency of "install\n" keyword.
IMO, add it to the first line of the ks file may work.

Reply:
ok, I had updated the code.

JOB ID     : 0fcf4cb3639c36f993efa20adf053c9e196c5184
JOB LOG    : /root/test_home/yhong/auto-command3/staf-kvm-devel/workspace/job-results/job-2017-09-18T05.23-0fcf4cb/job.log
 (1/4) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (3427.86 s)
 (2/4) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_threads: PASS (3985.28 s)
 (3/4) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (3273.76 s)
 (4/4) smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.ppc64le.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_threads: PASS (3364.55 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 14055.81 s
JOB HTML   : /root/test_home/yhong/auto-command3/staf-kvm-devel/workspace/job-results/job-2017-09-18T05.23-0fcf4cb/results.html
(workspace)[root@p8-host2 staf-kvm-devel]# python ConfigTest.py --testcase=unattended_install.cdrom.extra_cdrom_ks.multi_disk_install --driveformat=virtio_blk,virtio_scsi --guestname=RHEL.7.4 --platform=ppc64le --mem=8192 --vcpu=8 --verbose=no

JOB ID     : 0b43c9ba6f3cc733eacc75042d352381c19211c9
JOB LOG    : /home/yhong/staf-kvm-devel/workspace/job-results/job-2017-09-18T07.45-0b43c9b/job.log
 (1/4) smp_4.8145m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (3041.12 s)
 (2/4) smp_4.8145m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_threads: PASS (3061.20 s)
 (3/4) smp_4.8145m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_native: PASS (2998.93 s)
 (4/4) smp_4.8145m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_scsi.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.multi_disk_install.aio_threads: PASS (3015.46 s)
RESULTS    : PASS 4 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 12161.34 s
JOB HTML   : /home/yhong/staf-kvm-devel/workspace/job-results/job-2017-09-18T07.45-0b43c9b/results.html
(workspace)[root@dell-pe-fm120-1c staf-kvm-devel]# python ConfigTest.py --testcase=unattended_install.cdrom.extra_cdrom_ks.multi_disk_install --driveformat=virtio_blk,virtio_scsi --guestname=RHEL.7.4 --platform=x86_64 --mem=8192 --vcpu=8 --verbose=no

Reply:
The updated code : 
        if self.params.get("cmd_only_use_disk"):
            insert_info = self.params.get("cmd_only_use_disk") + '\n'
            contents += insert_info